tests/lib/devicetree/api/src/main.c

```diff
@@ -1625,6 +1625,13 @@ static void test_path(void)
 			     "/test/gpio@deadbeef"), "");
 }
 
+static void test_node_name(void)
+{
+	zassert_true(!strcmp(DT_NODE_FULL_NAME(DT_ROOT), "/"), "");
```

Comment:
Please add this or something similar, so we verify it works when there is no unit address:
```
	zassert_true(!strcmp(DT_NODE_FULL_NAME(TEST_TEMP),
			     "temperature-sensor"), "");
```

Reply:
Added